app/views/pages/home.html.erb

```diff
@@ -2,7 +2,7 @@
   <h1>Welcome to Vesta</h1>
   <p>
   Vesta is a web app that manages the housing process for Yale's residential colleges.
-  <% unless @current_college.name.empty? || @current_college.admin_email.empty? %>
+  <% unless !@current_college || @current_college.name.empty? || @current_college.admin_email.empty? %>
```

Comment:
This shouldn't change anything, the controller action is going to run regardless because it's a `before_action` in the controller.